podman.md

```diff
@@ -1,61 +1,110 @@
-# Running Hibernate Reactive tests on Podman
+# How to start the test databases using Podman
 
-As an alternative to Docker, you can use [podman] to start the databases used 
-by the test suite for Hibernate Reactive.
+The following sections will use [Podman][podman] to start the database of your choice, create the schema
+and create the credentials required to run the tests during the build.
+
+Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System.
+Containers can either be run as root or in rootless mode.
```

Comment:
A minor comment: `.md` files are supposed to be readable as text, so it's important to maintain a consistent line-length.

Reply:
ok